docs/user/whats-new.asciidoc

```diff
@@ -1,175 +1,144 @@
 
+image::images/discover-log-level.png[Log level badge displaying in the Discover grid]
 
 [discrete]
-==== Field statistics in ES|QL
+==== Recommended ES|QL queries
 
-Field statistics are now available in ES|QL. This feature is designed to provide comprehensive insights for each data field. With this enhancement, you can access detailed statistics such as distributions, averages, and other key metrics, helping you quickly understand your data. This makes data exploration and quality assessment more efficient, providing deeper insights and streamlining the analysis of field-level data in ES|QL.
+Writing ES|QL queries just got easier. Many users face challenges when authoring queries, and even more so when unfamiliar with the syntax or data structure. This can lead to inefficiencies in data analysis and visualization. We want to reduce the time it takes to create queries and to lower the learning curve for both new and existing users by suggesting recommended queries within the ES|QL Help menu and from the auto-complete.
 
-image::images/field-statistics-esql.png[Field statistics in ES|QL]
+image::images/esql-suggestions.png[A list of suggestions to get started with an ES|QL query, width=30%]
+_Recommended ES|QL queries from the ES|QL help menu_
 
-[discrete]
-==== Integrations support in the ES|QL editor when using FROM command.
+image::images/esql-autocomplete-suggestions.png[A list of suggestions in the autocomplete menu of an ES|QL query, width=50%]
+_Recommended ES|QL queries from auto-complete suggestions_
 
-We're excited to announce enhanced support for integrations in the ES|QL editor with the *FROM* command. Previously, you could only access indices, but now you can also view a list of installed integrations directly within the editor. This improvement streamlines your workflow, making it easier to manage and utilize various integrations while working with your data.
-
-image::images/integrations-in-esql.png[Accessing an integration from ES|QL]
 
 
 [discrete]
 === Dashboards
 
 [discrete]
-==== Field statistics in Dashboards
-
-It's now easier than ever to include your field statistics view from **Discover** into **Dashboards**. While running investigations, it is very common that you need to see some field information, such as unique values and their distribution, to make sense of the data. Select the fields that you want with your ES|QL query and get the document count, values, and distribution in your dashboard so you don't have to navigate back and forth to **Discover** to see this information.
+==== Manage dashboards more easily and efficiently
+As part of a series of improvements to help you find and manage your dashboards [started in version 8.15], we are adding a way to sort your dashboards by recently viewed (new default), an option to star your favorite dashboards, as well as some statistics to monitor the usage of your dashboards. 
```

Comment:
Rephrasing suggestion for better flow: "...[started in version 8.15], the new default way to sort your dashboards is by recently viewed, and we are adding an option to star..."

Reply:
you made me realize I forgot to add the link before the brackets -_- thanks